hazelcast/src/main/java/com/hazelcast/map/impl/eviction/Evictor.java

```diff
@@ -32,20 +32,21 @@
      * @param recordStore the recordStore
      * @return removal size.
      */
-    int findRemovalSize(RecordStore recordStore);
+    int findSizeToEvict(RecordStore recordStore);
```

Comment:
The javadoc doesn't explain at all what this method is doing and the method name either. So what is this method actually doing?